modules/power_z_unsupported_known_issues.adoc

```diff
@@ -0,0 +1,18 @@
+[id="power z known issues_{context}"]
+== IBM Z and LinuxONE known issues. 
+
+  * Only supported on {product-title} 4.6 
+
+== IBM Power Systems known issues.
+  
+  * Only supported on {product-title} 4.6
+
+== The following features are unsupported on IBM Z:
+
+** Jaeger streaming via AMQ/Kafka
```

Comment:
I would list this Jaeger restriction in the Jaeger Known Issues module (jaeger-rn-known-issues.adoc), right under the bullet where Apache Spark isn't supported.